### ISSUE TITLE
BUG: CUDA capability sm_86 error with pytorch

### ISSUE THREAD

**Issue by fMizki** [vocalpy/vak#508]
I'm on a computer with RTX3090, which is supported with "cuda" of which version is more than 11.0 and thus vak's "pytorch" version seems unmatched, so I get the warning;
```
NVIDIA GeForce RTX 3090 with CUDA capability sm_86 is not compatible with the current PyTorch installation.

The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_61 sm_70 sm_75 compute_37.
If you want to use the NVIDIA GeForce RTX 3090 GPU with PyTorch, please check the instructions at https://pytorch.org/get-started/locally/

  warnings.warn(incompatible_device_warn.format(device_name, capability, " ".join(arch_list), device_name))
```
I tried to update pytorch in (vak-env) which I created according to the installation documentation of vak to run vak with tweetynet, but I failed. It might be because my command was not correct as I'm not a skillful user, though.

**Comment by NickleDave** [comment]
Thank you @fMizki for reporting this bug.  

Could you please do the following:
1. before you make any further changes to your environment, can you save its current state to files and reply with those files as an attachment? I'd like to see the versions of libraries you have installed.

Please attach the two files you create by executing the commands below:
```
C:\Users\you> conda activate tweety-env
(tweety-env) C:\Users\you> conda env export > environment.yml
(tweety-env) C:\Users\you> conda list --explicit > spec-file.txt
```

My guess is that an earlier version got installed and that's what's causing the issue.

2. after you do that, please try making a *new* environment with the following commands, and then test it to see if you get the same error. (Don't remove the other environment yet in case we need it later!)
(You can use a different name besides `vak-env` like `vak-env-new-test` or something if you already have a `vak-env` environment)

```
conda create --name vak-env
conda install pytorch torchvision cudatoolkit=11.3 -c pytorch
conda install vak tweetynet -c conda-forge
```

Basically here you are making sure a more recent version of `pytorch` is installed before you install `vak`.

Let me know how that goes. Thanks

**Comment by fMizki** [comment]
> Could you please do the following:
> 
> 1. before you make any further changes to your environment, can you save its current state to files and reply with those files as an attachment? I'd like to see the versions of libraries you have installed.

Here are the files. I'm not sure the '.yml' file is shared properly, so please tell me if you couldn't open it.
[environment.zip](https://github.com/vocalpy/vak/files/8692099/environment.zip)

[spec-file.txt](https://github.com/vocalpy/vak/files/8692100/spec-file.txt)

> 2. after you do that, please try making a new environment with the following commands, and then test it to see if you get the same error. (Don't remove the other environment yet in case we need it later!)
> (You can use a different name besides vak-env like vak-env-new-test or something if you already have a vak-env environment)
> ```
> conda create --name vak-env
> conda install pytorch torchvision cudatoolkit=11.3 -c pytorch
> conda install vak tweetynet -c conda-forge
> ```

They worked!! Now I don't get the error I have got.
I didn't come up with your idea of installing version of `pytorch ` before vak.　

Thank you so much!!

**Comment by NickleDave** [comment]
Thank you @fMizki, that was perfect. I always forget you have to zip files when you attach them on GitHub--wish it was a little easier.

I see from the environment.yml that you had `pytorch==1.10.2` and `cudatoolkit=10.2.89`.

Could you by any chance tell me what version of `pytorch` you have in the new environment you created?  
I'm curious to know if it's `1.11`.

I need to figure out why older versions are getting installed by default -- that's not what I'd expected.  
But this is helpful

**Comment by fMizki** [comment]
I see. Thank you for telling me.

I got
`pytorch                 1.11.0          py3.10_cuda11.3_cudnn8_0    pytorch`

**Comment by NickleDave** [comment]
Great, thank you @fMizki !!! That helps to know.
And thanks for closing 🙂
